sirepo/job_supervisor.py

```diff
@@ -588,7 +588,7 @@ def __db_init_new(cls, data, prev_db=None):
             queueState="queued",
             canceledAfterSecs=None,
             computeJid=data.computeJid,
-            computeJobHash=data.computeJobHash,
+            computeJobHash=data.get("computeJobHash"),
```

Comment:
This is better than unused but it seems to me like an object here where if this field is accessed we blow up would be better. I worry about us comparing None's somewhere else and code working in an incorrect way.

Reply:
I can't fix this. computeJobHash=None is correct. There is no way to compute the job has, because there is not specification for the "job" since it is a method that does arbitrary things. The code in job_supervisor always compares with equals, and only for simulations, e.g. in downloadDataFile.

We are reusing "job" here for statelessCompute, and this is the complication. There might need to be a subclass SimulationJob, but that would be even more complicated.